src/reference-manual/deprecations.Rmd

```diff
@@ -123,6 +126,11 @@ target += foo_log(y, ...);
 functions or `_lpmf` for mass functions in the user-defined function.
 
 
+*Scheduled Removal*: Stan 2.32
+
+Note: Following Stan 2.32, users can stil define a function ending in `_log`,
+it will just no longer have a special meaning or support the `~` syntax.
+
 ## `lkj_cov` distribution
 
 *Deprecated*:The distribution `lkj_cov` is deprecated.
```

Comment:
There's a drop-in replacement for this.  I think it's just a lognormal prior on the scales with an LKJ prior on the correlations.  It's not a one-symbol for one-symbol replacement, though.  So probably not something that can be auto-handled by the pretty printer.

Reply:
I agree, it's just barely out-of-scope in my opinion